etc/test-lib.js

```diff
@@ -16,7 +16,7 @@
 'use strict';
 var valcheck = require('core-util-is');
 
-var mlutil    = require('../lib/mlutil.js');
+var mlutil = require('../lib/mlutil.js');
```

Comment:
One note - try to keep whitespace changes like this in a separate commit/PR, as it makes the actual changes much more difficult to see - not just now, but in the future when someone looks at this commit to see what was fixed.